syncmaster/scheduler/transfer_job_manager.py

```diff
@@ -47,6 +49,22 @@ def update_jobs(self, transfers: list[Transfer]) -> None:
                     args=(transfer.id,),
                 )
 
+    async def remove_orphan_jobs(self) -> None:
+        all_jobs = self.scheduler.get_jobs()
+        settings = self.settings
+        job_transfer_ids = [int(job.id) for job in all_jobs]
+
+        async with get_async_session(settings) as session:
+            result = await session.execute(
+                select(Transfer).where(Transfer.id.in_(job_transfer_ids)),
```

Comment:
```suggestion
                select(Transfer).where(Transfer.id = any_(job_transfer_ids)),
```
https://github.com/MobileTeleSystems/data-rentgen/blob/66fdd459bfc533136cc2bbd7de9496d1d8dc3307/data_rentgen/db/repositories/dataset.py#L48

This avoids generating new query for each number of ids, and instead passed list of ids as one argument of the query.